config/checkstyle/checkstyle.xml

```diff
@@ -124,7 +124,7 @@
         <!-- Checks for Size Violations.                    -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
         <module name="LineLength">
-            <property name="max" value="120"/>
```

Comment:
I have reverted checkstyle alteration, anyway, maybe it should be considered, if 120 characters is sufficient for nowadays displays.